src/utils/client.ts

```diff
@@ -353,18 +354,16 @@ export default class Client {
     return [trueSession, formData];
   }
 
-  submitAuthInfo({ authId, authPhone = '', authEmail = '' }: any) {
+  async submitAuthInfo({ authId, isStytchTemplateKey }: any) {
```

Comment:
I realized that we were passing in `isStytchTemplateKey`, but it wasn't actually being used by this fn..

Reply:
nice catch